doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1156,6 +1156,212 @@ Use the ``/settings`` API to enable or disable the enforcement of storage quotas
 
    curl -X PUT -d 'true' http://localhost:8080/api/admin/settings/:UseStorageQuotas
 
```

Comment:
The hierarchy of these docs seems a bit odd:

![Screenshot 2025-01-27 at 11 44 06 AM](https://github.com/user-attachments/assets/cb530419-f7ce-453c-8a08-bb8198804bdf)

Reply:
I agree. I have restructured it by deleting `Dataverse Collection Featured Items `section and reordering subsections.